.buildkite/pipeline.yml

```diff
@@ -828,6 +828,16 @@ steps:
         agents:
           slurm_mem: 20GB
 
+      - label: ":genie: Prognostic EDMFX Soares in a column"
+        command: >
+          julia --color=yes --project=examples examples/hybrid/driver.jl
+          --config_file $CONFIG_PATH/prognostic_edmfx_soares_column.yml
+          --job_id prognostic_edmfx_soares_column
+        artifact_paths: "prognostic_edmfx_soares_column/output_active/*"
+        agents:
+          slurm_mem: 20GB
+        soft_fail: true
```

Comment:
Do you need it to be soft fail?

Reply:
No - I removed this. Was copying what I'd done previously.